src/python/pants/option/global_options.py

```diff
@@ -329,6 +338,7 @@ def from_options(cls, options: OptionValueContainer) -> LocalStoreOptions:
     remote_store_rpc_retries=2,
     # Remote cache setup.
     remote_cache_eager_fetch=True,
+    remote_cache_warnings=RemoteCacheWarningsBehavior.first_only,
```

Comment:
should be ignore...
to my point in an earlier conversation, pants users don't care about cache errors when they are running their linters/formatters/pytest.
the same way you don't care about mypy or pytest cache when you are running those tools.